src/table/utils/handle-accessibility.ts

```diff
@@ -1,11 +1,31 @@
-export const getCellElement = (rootElement, cellCoord) =>
+import { stardust } from '@nebula.js/stardust';
+import { BaseSyntheticEvent } from 'react';
+
+interface ICellFocusProps {
+  focusType: string;
+  cell: HTMLTableCellElement | undefined;
+}
+interface IHandleResetFocusProps {
+  focusedCellCoord: [number, number];
+  rootElement: HTMLDivElement;
```

Comment:
i'm not sure this is always a div. Do you know? might be different depending on how you mount it

Reply:
should be `HTMLElement `